first pass at contributing.md (#2)
EarthProgram/use-cases: contributing.md

The documents in this repo are curated for quality and topicality. All on-topic, quality contributions will be considered.

## Issues
For most contributions, start from a Github issue. If you have an item for the community to consider for use cases or requirements, please open an issue in this repo, explaining the item to consider. It might be an entire new category of use cases or as simple as a typo. Or it might be an open invitation to solve a particular problem or to improve how we handle input.

Issues provide an opportunity for editors and other members of the community to consider your request before settling on a specific solution. Conversation occurs in comments on issues, giving different members of the community a chance to ask questions and discuss the topic. Some issues may be directed to other venues. Some may be resolved directly, e.g., by providing a concrete answer to a question. Other issues may develop a shared sense of how we can or should update the repository, these result in concrete Pull Requests, or PRs.

> To open an issue, simply click on the "Issues" tab [https://github.com/LegReq/use-cases/issues](issues) or directly click on [new issue](issues/new/choose). 

Reviewer: ```suggestion
To open an issue, simply click on the "Issues" tab [https://github.com/EarthProgram/use-cases/issues](issues) or directly click on [new issue](issues/new/choose). 
```

## Pull Requests
Once issues are mature enough to propose specific text changes, a Github Pull Request is the next step. Pull Requests provide specific changes to specific files, with the intention of eventually adopting and publishing those changes. PRs enable people to consider and discuss the text that will be adopted if the PR is accepted.

While PRs are under consideration, they may be further edited and updated as the community, hopefully, converges on a consensus for the update. Collaborators may directly propose edits on the PR, from minor typos to adding or removing entire sections.

> Ultimately, document editors retain editorial authority to choose the final text, subject to approval by the Earth Program board; however, the PR discussion is a forum for debate and argument that should guide the editors. For those documents managed by a work force or other collaborative team, editors should defer to the decision-making processes of that team.

Reviewer: ```suggestion
Ultimately, document editors who are approved through Earth Program governance retain editorial authority to choose the final text; however, the PR discussion is a forum for debate and argument that should guide the editors. For those documents managed by a work force or other collaborative team, editors should defer to the decision-making processes of that team.
```

Once a PR has run its course, it will either be accepted and become part of the work, or rejected and closed with explanation about the decision.

To open a Pull Request, first fork this repo, make your edits in the fork, then, from your fork, submit a pull request back to http://github.com/EarthProgram/use-cases

Best practices for PRs include
1. Create a new branch with your name and a unique phrase describing the intended update, such as `andrieu-fix-typos`. Submitting a PR from that particular branch will help others understand who is proposing the PR for what reason. 
2. Commit frequently and with good comments. Commits form the backbone of the git version tracking system. Good comments allow others to better track updates.
3. Use `git-status` and `git-diff` to remind yourself of your actual edits and help you write better commit messages. 
4. A commit per edit isn't necessary, but it can help clarify what changed. Highly granular commits can be a lifesaver when merge conflicts or other version management issues inevitably arise.
5. Provide a description of the changes in the PR itself.

## Questions

> If you have questions about how to contribute to this repo, please reach out to mailto:joe.andrieu@ixo.world or mailto:shaun.conway@ixo.world for assistance.

Reviewer: ```suggestion
If you have questions about how to contribute to this repo, please reach out to the Earth Program team at mailto:assistant@ixo.world for assistance.
```
